### DISCUSSION TITLE
How to set up giscus comments?

### DISCUSSION THREAD

**Discussion by TanYuanXiangElroy** [alshedivat/al-folio#2299] [Q&A] [answered]
I went to the Guscus  area in _config.yml line 138 and did its instruction of filling in the 

```
repo: 
repo_id: 
category: 
category_id:
```

But when I run the code on my local docker it shows

```
giscus comments misconfigured > Please follow instructions at
http://giscus.app/ and update your giscus configuration. {: .block-danger }
```
Is there something I missed out?

![image](https://github.com/alshedivat/al-folio/assets/89953102/f2775f80-06c1-4df1-aed1-c8c1e1f25fe6)

thanks for the help

**Comment by george-gca** [comment]
This is weird, specially this part: `{: .block-danger }`. Does this error occurs in any other pages?

**Reply by TanYuanXiangElroy** [reply, marked as answer]
Yes it appeared at other pages during the time of posting but guiscus comments works now.

I created another test page and it didn't work after I came back after 2 days it worked thanks for the help.
